src/store/api.ts

```diff
@@ -209,6 +209,16 @@ const getTreasuryPoolTokenAccountPK = multiAsync(async (connection: Connection,
 	return globalMarketStateData.treasuryPoolTokenAccount;
 });
 
+const getAssociatedUSDCTokenAddressPK = multiAsync(async (connection: Connection, wallet: Wallet) => {
```

Comment:
remove `getUserUSDCTokenAccount` as it's no longer needed now that we use the associated token account.